README.md

```diff
@@ -28,94 +28,17 @@ Decidim is a participatory democracy framework written on Ruby on Rails original
 
```

Comment:
Hahaha maaaaybe we should keep it simpler xD. Truth is they're all useful for one thing or another...